File: backend/api/src/main/scala/com/github/ai/split/api/response/PostMemberResponse.scala
package com.github.ai.split.api.response

import com.github.ai.split.api.GroupDto
import zio.json.{DeriveJsonEncoder, JsonEncoder}

case class PostMemberResponse(
  group: GroupDto
)

object PostMemberResponse {
  implicit val encoder: JsonEncoder[PostMemberResponse] = DeriveJsonEncoder.gen[PostMemberResponse]
}

File: backend/api/src/main/java/com/github/ai/split/api/response/PostMemberResponse.java
package com.github.ai.split.api.response;

import com.github.ai.split.api.GroupDto;
public class PostMemberResponse {

    public GroupDto group = new GroupDto();

    public PostMemberResponse() {}

    public PostMemberResponse(GroupDto group) {
        this.group = group != null ? group : new GroupDto();
    }
}

